src/main/resources/assets/twilightforest/lang/en_US.lang

```diff
@@ -51,6 +51,7 @@ tile.TFFireJet.12.name=Encased Popping Fire Jet
 tile.TFFireJet.13.name=Encased Flaming Fire Jet
 tile.TFNagastone.0.name=Nagastone Head
 tile.TFNagastone.1.name=Nagastone
+tile.TFNagastone.32767.name=Any Nagastone
```

Comment:
not sure if this is a good solution. I would guess its not more than 10 ids used so better add a propper way to add each block